.venv/bin/Get-GPPPassword.py

```
@@ -0,0 +1,321 @@
+                                else:
+                                    logging.debug("Found file %s" % sharedfile.get_longname())
+                except SessionError as e:
+                    logging.debug(e)
+            searchdirs = next_dirs
+            logging.debug("Next iteration with %d folders." % len(next_dirs))
+        return files
+
+    def parse_xmlfile_content(self, filename, filecontent):
+        results = []
+        try:
+            root = minidom.parseString(filecontent)
```

Ruff (0.14.2)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S318)